src/LibraryManagement.Application/CommandHandlers/Authentication/CreateUserCommandHandler.cs

```diff
@@ -0,0 +1,49 @@
+﻿using LibraryManagement.Application.Commands.Authentication;
+using LibraryManagement.Domain.Entities.Authentication;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using System.ComponentModel.DataAnnotations;
+
+namespace LibraryManagement.Application.CommandHandlers.Authentication;
+
+public class CreateUserCommandHandler : IRequestHandler<CreateUserCommand, Guid>
+{
+    private readonly UserManager<User> userManager;
```

Comment:
You should add the "_" symbol before a variable

Reply:
Updated